docs/pages/api-reference/liveblocks-emails.mdx

```diff
@@ -6,111 +6,840 @@ meta:
+**We’ll only show the highlighted part below**, as it’s assumed you’ve set this
+already, and your file contains `liveblocks` and `event`.
 
-Then the functions will return either all comments in the thread **created
-after** the `readAt`date of the notifucation **and created before or at the same
-date** of `notifiedAt` date of the notification. Or either it will return all
-comments in the thread **created before or at the same date** of `notifiedAt`
-date of the notification if the notification is not read by the user who has
-recieved the inbox noitification.
+## Thread notification emails [#thread-notification-emails]
 
-#### Last unread mention [#last-unread-mention]
+These functions help you create emails to notify users of _unread comments_ in
+threads. They fetch each relevant comment, filtering out any that have already
+been read, and help you style each comment’s body with either
+[React](#prepare-thread-notification-email-as-react) or
+[HTML](#prepare-thread-notification-email-as-html).
 
-The last unread mention is computed based on the list of the unread replies by
-checking if the user who recieved the inbox notification has been mentionned in
-one of those comments and will return the last one (newest created comment) or
-`null` if it's not the case.
+<Figure>
+  <Image
+    src="/assets/emails/new-comments-email.png"
+    alt="New comments email"
+    width={1567}
+    height={830}
+  />
+</Figure>
 
-### prepareThreadNotificationAsReact [#prepare-thread-notification-as-react]
+The function also helps you distinguish between _unread mentions_ and _unread
+replies_.
 
-Prepares data from a
-[`ThreadNotificationEvent`](/docs/platform/webhooks#Thread-notification) and
-convert comment bodies as [React](https://react.dev/) nodes.
+- A thread has _unread replies_ if a comment was created after the `readAt` date
+  on the notification, and created before or at the same time as the
+  `notifiedAt` date. All unread replies are returned in an array.
+- A thread has an _unread mention_ if it has unread replies, and one reply
+  mentions the user. A single comment with the latest mention is returned.
 
-It returns a `ThreadNotificationEmailDataAsReact` object or `null` if there are
-no unread comments.
+The `type` property that’s returned lets you know which this notification is
+for.
+
+```js
+{
+  type: "unreadReplies",
+  // ...
+}
+```
+
+### prepareThreadNotificationEmailAsReact [#prepare-thread-notification-email-as-react]
+
+Takes a
+[thread notification webhook event](/docs/platform/webhooks#Thread-notification)
+and returns unread comment body(s) related to the notification, as React nodes.
+It can return one of three formats, an `unreadMention` type containing one
+comment, an `unreadReplies` type returning multiple comments, or `null` if there
+are no unread mentions/replies. You can also
+[resolve user & room data](#prepare-thread-notification-email-as-react-resolving-data)
```

Comment:
```suggestion
[resolve users & room data](#prepare-thread-notification-email-as-react-resolving-data)
```

Reply:
Another English thing! This is short for `resolve user data & room data`, which both work as a singular, despite multiple users 👍